api/python/cellxgene_census/src/cellxgene_census/experimental/ml/pytorch.py

```diff
@@ -256,6 +295,7 @@ def __init__(
         batch_size: int = 1,
         sparse_X: bool = False,
         soma_buffer_bytes: Optional[int] = None,
+        is_eager_fetch: bool = True,
```

Comment:
Nit: don't love the parameter name given that it is a directive to use eager.  Suggest something like:
* use_eager_fetch
* be_eager
etc.

Reply:
this was resolved after a conversation with Pablo.